plugins/connection/docker_api.py

```diff
@@ -164,7 +164,7 @@ def exec_command(self, cmd, in_data=None, sudoable=False):
             stderr=True,
             stdin=need_stdin,
             user=self._play_context.remote_user or '',
-            workdir=None,
+            # workdir=None,  - only works for Docker SDK for Python 3.0.0 and later
```

Comment:
shouldn't it be deprecated instead?

Reply:
No, right now this isn't used at all (`None` is the default value anyway, which means "use whatever working dir the container is using anyway"). Having this always present only breaks usage wiht Docker SDK for Python < 3.0.0, but doesn't have any effect on >= 3.0.0.